components/sankey/overview.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Overview
+page_title: Sankey Overview
+description: Overview of the Sankey diagram for Blazor.
+slug: sankey-overview
+tags: telerik,blazor,sankey,diagram,chart,overview
+published: True
+position: 0
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Blazor Sankey Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/sankey-diagram" target="_blank">Blazor Sankey diagram component</a> allows you to visualize changing flows and their distribution between domains. Sankey diagrams suit a variety of use cases like the representation of website traffic, budget breakdowns, energy flow, and others.
+
+## Creating Sankey Diagram
+
+1. Add the `<TelerikSankey>` tag to your razor page.
+1. [Bind the Sankey Data]({%slug sankey-data-binding%}).
```

Comment:
Can we make this step more valuable and detailed, so that it doesn't just point to another article? It won't be possible to explain everything in a few sentences, but let's just prepare the reader with the fundamentals. For example, consider something like "The Sankey Diagram expects its data to have a specific structure and use specific object types."